promlog/log.go

```diff
@@ -26,14 +26,17 @@ import (
 	"github.com/go-kit/log/level"
 )
 
+// This timestamp format differs from RFC3339Nano by using .000 instead
+// of .999999999 which changes the timestamp from 9 variable to 3 fixed
+// decimals (.130 instead of .130987456).
```

Comment:
I think this comment needs to go back where it was.

Reply:
Fixed! I need to get more sleep 😃